Example/Backpack/Stories/HorizontalNavigationStory.swift

```diff
@@ -0,0 +1,83 @@
+//
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright © 2021 Skyscanner Ltd. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+
+enum HorizontalNavigationStory: String, StoryGroup {
+    case `default` = "Default"
+    case small = "Small"
+    case withoutUnderline = "Without underline"
+    case withIcons = "With icons"
+    case smallWithIcons = "Small with icons"
+    case wide = "Wide"
+    case withScroll = "With sroll"
+    case customItems = "Custom items"
+    case withNotifications = "With notifications"
+    case withBadge = "With badge"
+    case alternate = "Alternate"
+
+    var title: String {
+        self.rawValue
+    }
+
+    var presentableStory: Presentable {
+        let storyboard = loadStoryboard(name: "Main")
+        let viewController = storyboard("HorizontalNavViewController").makeViewController
+
+        return CustomPresentable(generateViewController: enrich(viewController, self.configureStory))
+    }
+
+    // This isn't actually complex, it's just a large switch
```

Comment:
Having a large number of different paths through software is the definition of cyclomatic complexity. I think in this case `M=13`.
So this comment is a lie.

Reply:
Yes, but cyclomatic complexity is a bit flawed in this way it tries to objectively measure complexity but I think this method subjectively isn't complex. Any change I could make to this function would not make it less complex or easier to read